openwisp_radius/api/utils.py

```diff
@@ -15,6 +17,20 @@ def _get_error_dict(self, error):
         return dict_
 
 
+class IDVerificationHelper(object):
+    def _needs_identity_verification(self, org):
```

Comment:
instead of org we could pass the keyword arguments to filter the org,
then we could use the following queryset:

```python
Organization.objects.select_related(
    'radius_settings'
).get(**organization_filter_kwargs)
```

Reply:
Done (https://github.com/openwisp/openwisp-radius/pull/237/commits/ef3686e4dbe8c5e85a5bec1769da36eaafd87ea8)